src/repository/models.py

```diff
@@ -623,6 +632,52 @@ def local_url(self):
 
         return url
 
+    def create_article(self, journal, workflow_stage, journal_license, journal_section, force=False):
```

Comment:
Can we have a Unit Test for this function please (with and without `force` behaviour)